test/es-module/test-typescript.mjs

```diff
@@ -3,6 +3,31 @@ import * as fixtures from '../common/fixtures.mjs';
 import { match, strictEqual } from 'node:assert';
 import { test } from 'node:test';
 
+test('expect process.features.typescript to be \'strip\' when --experimental-strip-types', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--no-warnings',
+    '--experimental-strip-types',
+    fixtures.path('typescript/echo-process-features-typescript.cjs'),
+  ]);
+
+  strictEqual(result.stderr, '');
+  strictEqual(result.stdout, process.config.variables.node_use_amaro ? 'strip\n' : 'false\n');
+  strictEqual(result.code, 0);
+});
+
+test('expect process.features.typescript to be \'transform\' when --experimental-transform-types', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--no-warnings',
+    '--experimental-transform-types',
+    fixtures.path('typescript/echo-process-features-typescript.cjs'),
+  ]);
+
+  strictEqual(result.stderr, '');
+  strictEqual(result.stdout, process.config.variables.node_use_amaro ? 'transform\n' : 'false\n');
+  strictEqual(result.code, 0);
+});
```

Comment:
@marco-ippolito I guess if this lands then https://github.com/nodejs/node/pull/55332/ would need to either move the tests back to after the skip or explicitly check for the error introduced there?

Reply:
That's not my understanding, no. Why would we skip those tests?